2022-05-30 23:16:39,316 INFO    Thread-2 (_run_job):91691 [wandb_setup.py:_flush():75] Loading settings from /home/emili/.config/wandb/settings
2022-05-30 23:16:39,316 INFO    Thread-2 (_run_job):91691 [wandb_setup.py:_flush():75] Loading settings from wandb/settings
2022-05-30 23:16:39,316 INFO    Thread-2 (_run_job):91691 [wandb_setup.py:_flush():75] Loading settings from environment variables: {'api_key': '***REDACTED***', 'entity': 'emilibonet', 'project': 'gar_itw', 'root_dir': '/home/emili/Documents/Universitat/TFG/cv-gender-age-recognition', 'sweep_id': '0bg14jeo', 'run_id': 'g7tk99pc', 'sweep_param_path': '/home/emili/Documents/Universitat/TFG/cv-gender-age-recognition/wandb/sweep-0bg14jeo/config-g7tk99pc.yaml'}
2022-05-30 23:16:39,316 INFO    Thread-2 (_run_job):91691 [wandb_setup.py:_flush():75] Inferring run settings from compute environment: {'program_relpath': 'code/models/ours/sweeper.py', 'program': '/home/emili/Documents/Universitat/TFG/cv-gender-age-recognition/code/models/ours/sweeper.py'}
2022-05-30 23:16:39,316 INFO    Thread-2 (_run_job):91691 [wandb_init.py:_log_setup():437] Logging user logs to /home/emili/Documents/Universitat/TFG/cv-gender-age-recognition/wandb/run-20220530_231639-g7tk99pc/logs/debug.log
2022-05-30 23:16:39,316 INFO    Thread-2 (_run_job):91691 [wandb_init.py:_log_setup():438] Logging internal logs to /home/emili/Documents/Universitat/TFG/cv-gender-age-recognition/wandb/run-20220530_231639-g7tk99pc/logs/debug-internal.log
2022-05-30 23:16:39,316 INFO    Thread-2 (_run_job):91691 [wandb_init.py:init():471] calling init triggers
2022-05-30 23:16:39,316 INFO    Thread-2 (_run_job):91691 [wandb_init.py:init():474] wandb.init called with sweep_config: {'batch_size': 14, 'dropout': 0.49144368502225866, 'lr': 0.001, 'momentum': 0.7129512937678939}
config: {}
2022-05-30 23:16:39,316 INFO    Thread-2 (_run_job):91691 [wandb_init.py:init():524] starting backend
2022-05-30 23:16:39,316 INFO    Thread-2 (_run_job):91691 [backend.py:_multiprocessing_setup():97] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-05-30 23:16:39,321 INFO    Thread-2 (_run_job):91691 [backend.py:ensure_launched():217] starting backend process...
2022-05-30 23:16:39,322 INFO    Thread-2 (_run_job):91691 [backend.py:ensure_launched():222] started backend process with pid: 91731
2022-05-30 23:16:39,322 INFO    Thread-2 (_run_job):91691 [wandb_init.py:init():533] backend started and connected
2022-05-30 23:16:39,326 INFO    Thread-2 (_run_job):91691 [wandb_run.py:_config_callback():1131] config_cb None None {'batch_size': 14, 'dropout': 0.49144368502225866, 'lr': 0.001, 'momentum': 0.7129512937678939}
2022-05-30 23:16:39,327 INFO    Thread-2 (_run_job):91691 [wandb_init.py:init():597] updated telemetry
2022-05-30 23:16:39,331 INFO    Thread-2 (_run_job):91691 [wandb_init.py:init():628] communicating run to backend with 30 second timeout
2022-05-30 23:16:40,426 INFO    Thread-2 (_run_job):91691 [wandb_run.py:_on_init():1923] communicating current version
2022-05-30 23:16:40,843 INFO    Thread-2 (_run_job):91691 [wandb_run.py:_on_init():1927] got version response upgrade_message: "wandb version 0.12.17 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-05-30 23:16:40,843 INFO    Thread-2 (_run_job):91691 [wandb_init.py:init():659] starting run threads in backend
2022-05-30 23:16:45,427 INFO    Thread-2 (_run_job):91691 [wandb_run.py:_console_start():1897] atexit reg
2022-05-30 23:16:45,427 INFO    Thread-2 (_run_job):91691 [wandb_run.py:_redirect():1770] redirect: SettingsConsole.REDIRECT
2022-05-30 23:16:45,428 INFO    Thread-2 (_run_job):91691 [wandb_run.py:_redirect():1775] Redirecting console.
2022-05-30 23:16:45,428 ERROR   Thread-2 (_run_job):91691 [wandb_run.py:_redirect():1834] Failed to redirect.
Traceback (most recent call last):
  File "/home/emili/miniconda3/envs/viume/lib/python3.10/site-packages/wandb/sdk/wandb_run.py", line 1827, in _redirect
    out_redir.install()
  File "/home/emili/miniconda3/envs/viume/lib/python3.10/site-packages/wandb/sdk/lib/redirect.py", line 709, in install
    _WSCH.add_fd(self._pipe_read_fd)
  File "/home/emili/miniconda3/envs/viume/lib/python3.10/site-packages/wandb/sdk/lib/redirect.py", line 659, in add_fd
    self._register()
  File "/home/emili/miniconda3/envs/viume/lib/python3.10/site-packages/wandb/sdk/lib/redirect.py", line 644, in _register
    old_handler = signal.signal(signal.SIGWINCH, lambda *_: None)
  File "/home/emili/miniconda3/envs/viume/lib/python3.10/signal.py", line 56, in signal
    handler = _signal.signal(_enum_to_int(signalnum), _enum_to_int(handler))
ValueError: signal only works in main thread of the main interpreter
2022-05-30 23:16:45,429 INFO    Thread-2 (_run_job):91691 [wandb_init.py:init():684] run started, returning control to user process
2022-05-30 23:16:45,429 INFO    Thread-2 (_run_job):91691 [wandb_run.py:_finish():1685] finishing run emilibonet/gar_itw/g7tk99pc
2022-05-30 23:16:45,429 INFO    Thread-2 (_run_job):91691 [wandb_run.py:_atexit_cleanup():1866] got exitcode: 1
2022-05-30 23:16:45,430 INFO    Thread-2 (_run_job):91691 [wandb_run.py:_restore():1838] restore
2022-05-30 23:16:45,852 INFO    Thread-2 (_run_job):91691 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  total_bytes: 940
}

2022-05-30 23:16:46,379 INFO    Thread-2 (_run_job):91691 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 940
  total_bytes: 940
}

2022-05-30 23:16:46,480 INFO    Thread-2 (_run_job):91691 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 940
  total_bytes: 13554
}

2022-05-30 23:16:46,582 INFO    Thread-2 (_run_job):91691 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 940
  total_bytes: 13554
}

2022-05-30 23:16:46,685 INFO    Thread-2 (_run_job):91691 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 940
  total_bytes: 13554
}

2022-05-30 23:16:46,787 INFO    Thread-2 (_run_job):91691 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 940
  total_bytes: 13554
}

2022-05-30 23:16:46,891 INFO    Thread-2 (_run_job):91691 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 940
  total_bytes: 13554
}

2022-05-30 23:16:46,994 INFO    Thread-2 (_run_job):91691 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 940
  total_bytes: 13554
}

2022-05-30 23:16:47,096 INFO    Thread-2 (_run_job):91691 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 13554
  total_bytes: 13554
}

2022-05-30 23:16:47,199 INFO    Thread-2 (_run_job):91691 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 13554
  total_bytes: 13554
}

2022-05-30 23:16:47,303 INFO    Thread-2 (_run_job):91691 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 13554
  total_bytes: 13554
}

2022-05-30 23:16:47,406 INFO    Thread-2 (_run_job):91691 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 13554
  total_bytes: 13554
}

2022-05-30 23:16:47,508 INFO    Thread-2 (_run_job):91691 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 13554
  total_bytes: 13554
}

2022-05-30 23:16:47,612 INFO    Thread-2 (_run_job):91691 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 13554
  total_bytes: 13554
}

2022-05-30 23:16:48,349 INFO    Thread-2 (_run_job):91691 [wandb_run.py:_on_finish():1995] got exit ret: file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 13554
  total_bytes: 13554
}

2022-05-30 23:16:49,309 INFO    Thread-2 (_run_job):91691 [wandb_run.py:_on_finish():1995] got exit ret: done: true
exit_result {
}
file_counts {
  wandb_count: 5
}
pusher_stats {
  uploaded_bytes: 13554
  total_bytes: 13554
}
local_info {
}

2022-05-30 23:16:50,606 INFO    Thread-2 (_run_job):91691 [wandb_run.py:_footer_history_summary_info():3102] rendering history
2022-05-30 23:16:50,606 INFO    Thread-2 (_run_job):91691 [wandb_run.py:_footer_history_summary_info():3134] rendering summary
2022-05-30 23:16:50,607 INFO    Thread-2 (_run_job):91691 [wandb_run.py:_footer_sync_info():3057] logging synced files
